karapace/schema_reader.py

```diff
@@ -247,11 +248,13 @@ def run(self) -> None:
                     self.offset = self._get_beginning_offset()
                 try:
                     self.handle_messages()
+                    self.consecutive_unexpected_errors = 0
                 except ShutdownException:
                     self._stop_schema_reader.set()
                     shutdown()
                 except Exception as e:  # pylint: disable=broad-except
                     self.stats.unexpected_exception(ex=e, where="schema_reader_loop")
+                    self.consecutive_unexpected_errors += 1
```

Comment:
We only care about the message handling errors? This won't consider other possible kafka errors.

Reply:
I would assume if there are kafka problems they would show up in consume call